lib/cartopy/examples/barbs.py

```diff
@@ -6,6 +6,7 @@
 
 
 def main():
+    fig = plt.figure(figsize=(10, 5))
```

Comment:
Although I agree with this change, I don't understand why you choose to add the figsize to only these particular example plots since there are many examples that don't set the figsize (e.g. the [arrows example](http://scitools.org.uk/cartopy/docs/latest/examples/arrows.html)).